docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    with tempfile.TemporaryDirectory() as tmp_dir:
</pre><p>I'd appreciate a comment on what is being done here:</p><p><code># Let's construct a file_list text file, which contains the names of the data files to be loaded</code></p><p> or similar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>